src/lib/storage/chunk.cpp

```diff
@@ -14,27 +14,25 @@
 
 namespace opossum {
 
-void Chunk::add_segment(std::shared_ptr<BaseSegment> segment) {
-  // Implementation goes here
-}
+void Chunk::add_segment(std::shared_ptr<BaseSegment> segment) { _columns.push_back(segment); }
 
 void Chunk::append(const std::vector<AllTypeVariant>& values) {
-  // Implementation goes here
-}
+  DebugAssert(_columns.size() == values.size(), "The number of passed values doesn't match the number of columns.");
 
-std::shared_ptr<BaseSegment> Chunk::get_segment(ColumnID column_id) const {
-  // Implementation goes here
-  return nullptr;
+  for (auto column_id = 0ul; column_id < _columns.size(); column_id++) {
```

Comment:
I would recommend 
```
for (ColumnID current_column_id{0}; current_column_id < column_count(); ++current_column_id)
    _segments[current_column_id]->append(values[current_column_id]);
```
since it incorporates the ColumnID type.